webapp/server/methods.js

```diff
@@ -335,4 +335,36 @@ Meteor.methods({
       }
     });
   },
+
+  // Get the genes that should have icons appear next to their names.
+  // They are contained in a special geneSetCollection named
+  // "GeneSets appearing as icons. Do not delete. oRZvz3Gbim"
+  // To specify the icon & color, each gene_set's name should be, eg, "yellow star".
+  // (this is a meteor method & not a publication so we can transform the find server-side)
+  getGeneInfos(){
+    let findBlessedSet = {"name":"GeneSets appearing as icons. Do not delete. oRZvz3Gbim"};
+
+    let addExtraInfo = function(geneSet) {
```

Comment:
How? Using the low-level interface in http://docs.meteor.com/api/pubsub.html#Meteor-publish ? I can't find docs for it now but my understanding is that you can't use `transform` on server-side publications, only client-side.